samples/net/sockets/dumb_http_server/src/socket_dumb_http.c

```diff
@@ -24,7 +24,7 @@
 #endif
 
 static const char content[] = {
-#if 0
+#if 1
```

Comment:
use a macro here instead, then we can set this in testcase.yaml for testing different loads.

Reply:
Changing the default here also impacts the documentation (https://www.zephyrproject.org/doc/samples/net/sockets/dumb_http_server/README.html) that says what the output looks like when you run the sample.